java/code/src/com/redhat/rhn/manager/content/MgrSyncUtils.java

```diff
@@ -126,6 +126,9 @@ public static ChannelArch getChannelArch(PackageArch packageArch, String channel
         else if (arch.equals("ppc64")) {
             arch = "ppc";
         }
+        else if (arch.equals("amd64")) {
+            arch = "amd64-deb";
```

Comment:
why not only `amd64`?

Reply:
amd64 is a RPM architecture in spacewalk code

Reply:
Ok, SCC -> map "amd64" to "amd64-deb" and repo type "deb"